### DISCUSSION TITLE
Where can I find the example.env

### DISCUSSION THREAD

**Discussion by logantai24** [directus/directus#13653] [Archive]
After recent update on directus.io docs, the link to example.env [here](https://docs.directus.io/self-hosted/installation/manual/#_3-setup-a-configuration-file) is broken. Anyone knows where the example.env can be found? Thanks for the help🙏

**Comment by azrikahar** [comment]
Good catch! The linked example.env was removed in #13352 in favor of consolidating it over here: https://github.com/directus/directus/blob/main/api/src/cli/utils/create-env/env-stub.liquid

**Reply by benhaynes** [reply]
Do we need to update that link in the docs so it doesn't throw a 404?

**Reply by azrikahar** [reply]
@sxsws is on it with #13656! (I believe they did comment here about it and subsequently open the PR, but it seems like they deleted the comment here 😄)

**Reply by sxsws** [reply]
I did, but deleted the comment as I thought it was unnecessary ;) The PR should be open?

**Reply by rijkvanzanten** [reply]
PR is already merged 🏎💨
